### DISCUSSION TITLE
Don't work with blogger?

### DISCUSSION THREAD

**Discussion by 7rainwr3ck** [biati-digital/glightbox#471] [General]
I can't get it to work with blogger, I've included some code to add the "glighbox" class
I include cdn https://cdn.jsdelivr.net/npm/glightbox/dist/css/glightbox.min.css
https://cdn.jsdelivr.net/gh/mcstudios/glightbox/dist/js/glightbox.min.js

and the code

<script type="text/javascript">
   const lightbox = GLightbox({ ...options });
</script>

Unfortunately glightbox doesn't work, I kept the script in a test page if you want to take a look at the source. Maybe there is some incompatibility with other scripts? Here is the link https://www.sicksadworld.it/p/test_12.html

_Originally posted by @7rainwr3ck in https://github.com/biati-digital/glightbox/discussions/469_

**Comment by gingerchew** [comment]
The options variable you're using inside the object isn't defined.

**Comment by 7rainwr3ck** [comment]
Okay, that was stupid lol.  I thought it would go by default even without settings.  I think I missed this passage due to my bad english.  Now it works and I love it, thanks!
